iosApp/iosApp/Pages/Onboarding/OnboardingScreenView.swift

```diff
@@ -23,19 +23,29 @@ struct OnboardingScreenView: View {
     @Environment(\.dynamicTypeSize) var typeSize
     @State private var pulse: CGFloat = 1
 
-    private var haloOffset: CGFloat {
-        let height = UIScreen.current?.bounds.height ?? 852.0
-        return -((height / 2) - (height / 3) + 2)
-    }
+    private var screenHeight: CGFloat { UIScreen.current?.bounds.height ?? 852.0 }
+    private var screenWidth: CGFloat { UIScreen.current?.bounds.width ?? 393.0 }
 
-    private var locationHaloSize: CGFloat {
-        let width = UIScreen.current?.bounds.width ?? 393.0
-        return width * 0.8
-    }
+    // Use less padding on smaller screens
+    private var bottomPadding: CGFloat { screenHeight < 812 ? 16 : 52 }
+    private var sidePadding: CGFloat { screenWidth < 393 ? 16 : 32 }
+
+    private var locationHaloSize: CGFloat { screenWidth * 0.8 }
+    private var moreHaloSize: CGFloat { screenWidth * 0.55 }
 
-    private var moreHaloSize: CGFloat {
-        let width = UIScreen.current?.bounds.width ?? 393.0
-        return width * 0.55
+    private var haloOffset: CGFloat {
+        let height = screenHeight
+        let width = screenWidth
+        // If the aspect ratio of the screen is greater than the image,
+        // it will be cut off on the top and bottom, which changes the
+        // position of the icon we want the halo to be centered on
+        // (390x844 are the background image dimensions)
+        if width / height >= 390 / 844 {
+            let scaledHeight = (width / 390) * 844
+            return -((scaledHeight / 2) - (scaledHeight / 3) - 6)
```

Comment:
@BrandonTR Thanks for catching the scaling issues! Here's what it looks like now on a 13 mini and an SE (3rd gen). The overlapping with the route line isn't ideal, but I think this is fine for now. The main issue was that the ZStack with the background and page content was overlapping the bounds of the screen, so I moved all the background images into `.background`s, then also shrunk the padding on smaller screen sizes to allow more room for the text. Then I also updated the halo offset function here to do this janky thing (I have no idea where the extra 6 pixels came from). It looks good enough, but isn't perfectly aligned on all devices, but at most it should only be a few pixels off and isn't really noticeable. 

![Simulator Screenshot - iPhone 13 mini - 2024-10-31 at 19 22 15](https://github.com/user-attachments/assets/484b6e4d-d7e4-48b3-987e-effa6bbf429a) ![Simulator Screenshot - iPhone SE (3rd generation) - 2024-10-31 at 19 23 07](https://github.com/user-attachments/assets/3e92ebe7-17e4-47d3-b970-b9e3e9e4297e)